### ISSUE TITLE
Scan of multiple hosts fails

### ISSUE THREAD

**Issue by weddige** [nabla-c0d3/sslyze#429]
**Describe the bug**
Sslyze does not scan multiple hosts in any case.

**To Reproduce**
1. Install SSLyze 3.0.1
2. Run `sslyze.exe --regular google.de google.com`
4. The results contain only scan results for one host:

```
> sslyze.exe --regular google.de google.com

 CHECKING HOST(S) AVAILABILITY
 -----------------------------

   google.de:443                       => 172.217.18.99
   google.com:443                       => 172.217.21.238




 SCAN RESULTS FOR GOOGLE.DE:443 - 172.217.18.99
 ----------------------------------------------

 * OpenSSL Heartbleed:
                                          OK - Not vulnerable to Heartbleed
...
```
(In this case ` SCAN RESULTS FOR GOOGLE.COM:443` was missing)

This does not always happen, but I could reproduce it with `google.de` and `google.com`. With other domains it worked fine.

**Expected behavior**
All hosts are scanned.

**Python environment (please complete the following information):**
 - OS: Windows 10
 - Python version: 3.8

**Additional context**
I didn't investigate the issue further yet. More information may follow in the next days if necessary.

**Comment by nabla-c0d3** [comment]
Thanks for the report

**Comment by nabla-c0d3** [comment]
Fix released in v3.0.2.
